tools/isobuild/bundler.js

```diff
@@ -1223,13 +1223,16 @@ class Target {
           continue;
         }
 
+        var data = await resource.data;
+        var hash = await resource.hash;
+
         const fileOptions = {
           info: 'unbuild ' + resource,
           arch: this.arch,
-          data: await resource.data,
+          data,
           cacheable: false,
-          hash: await resource.hash,
-          skipSri: !!await resource.hash
+          hash,
+          skipSri: !!hash
```

Comment:
This hasn’t had an impact, as `resource.data` and `resource.hash` are never promises, at least based on my debugging. I'm unsure why we are using `await` here, possibly due to an edge case I haven’t identified. I refactored the code to reduce the number of await calls for the same variables.